scripts/es-disk-usage

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+
+cd $(dirname "${BASH_SOURCE[0]}")
```

Comment:
My bash fu is not great but won't this leave you in `this` directory at the end of the run.  Might it be better to have something like `source $(dirname "${BASH_SOURCE[0]}")/env-prop`

Reply:
nope, tested locally and this doesn't move the caller script to a different folder, as this is a subshell